mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.kt

```diff
@@ -1042,7 +1054,7 @@ class MainActivity : AbstractBaseActivity(), ConnectionFactory.UpdateListener {
     }
 
     private fun showMissingPermissionsWarningIfNeeded() {
-        val missingPermissions = BackgroundTasksManager.KNOWN_KEYS
+        var missingPermissions = BackgroundTasksManager.KNOWN_KEYS
```

Comment:
Instead of making this `var`, I'd rather keep it as `val`, but as `MutableList` (basically remove line 1067), and to the conversion to `Array` later (in line 1101).